packages/ordeq-iceberg/pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["setuptools>=75.0", "setuptools_scm>=8.1.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ordeq_iceberg"
+dynamic = ["version"]
+readme = "README.md"
+license = "MIT"
+license-files = ["LICENSE"]
+authors = [{ name = "Jose R. Sanchez Viloria <@josersanvil>" }]
+maintainers = [{ name = "Jose R. Sanchez Viloria <@josersanvil>" }]
+requires-python = ">=3.10"
+dependencies = ["ordeq>=1.3.3", "pyiceberg[sql-sqlite]>=0.10.0"]
+
```

Comment:
The ordeq-devs section is missing here. Check the other pyproject.toml files for examples.